base_group_backend/README.rst

```diff
@@ -94,37 +74,21 @@ that check if user is part of the `base.group_user` or `share == False` group.
 Configuration
 =============
 
-To allow `group_backend` to interact with a model you can either add access rules to the group
-or you can add `implied_ids` to `group_backend`.
-
-.. note::
+To allow a user from the ``group_backend_ui_users`` group to interact with a specific model you can either add an access rules to this model for ``group_backend_ui_users`` or you can add ``group_backend_ui_users`` to the ``implied_ids`` of a new specific group.
 
-   Be aware users can only belong to one group from the user type category
-   (`base.module_category_user_type`). So your other groups can't inherit both
-   internal users and backend users.
+The Backend groups are from the "User types" category (``base.module_category_user_type``), the same category as "Internal User" (``base.group_user``), "Portal" (``base.group_portal``) or Public (``base.group_public``). Be aware that a user can only belongs to **one group of this category**.
 
 Usage
 =====
 
-To use this module, you need to:
-
-#. Go to Configuration / Users / Users, choose a user and set the user type.
-
-You get a users that is only able to access to the Odoo backend which you
-can attach other groups that not implies other kind of users (`portal`,
-`internal users`)
+To use this module, you need can just add a user to the group "Backend user" or "Backend UI user" through the user's form page.
```

Comment:
maybe choose one of them or none: need/can

```suggestion
To use this module, add a user to the group "Backend user" or "Backend UI user" through the user's form page.
```

Reply:
sorry for the suggestion in the wrong file :-/

Reply:
Thanks for the review! Done!